pkg/util/hw_helper.go

```diff
@@ -0,0 +1,39 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2018 Red Hat, Inc.
+ *
+ */
+
+package util
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const PCI_ADDRESS_PATTERN = `^([\da-fA-F]{4}):([\da-fA-F]{2}):([\da-fA-F]{2}).([0-7]{1})$`
+
+func ParsePciAddress(pciAddress string) ([]string, error) {
```

Comment:
Please add a documentation for this function, what does it return? Maybe it would be better to return (string, string, string, string, error) and give it proper names.

Reply:
I'd prefer to return the virt-launcher Address struct here, but that creates a circular dependency. If it's not a struct then I'd prefer a list of PCI DSBF fields. I'll add a comment.